crates/bifrost/src/providers/replicated_loglet/sequencer/mod.rs

```diff
@@ -0,0 +1,15 @@
+use restate_core::network::NetworkError;
+use restate_types::nodes_config::NodesConfigError;
+
+mod node;
+
+#[derive(thiserror::Error, Debug)]
+enum Error {
+    #[error("invalid node-set configuration")]
+    InvalidNodeSet,
+
+    #[error(transparent)]
+    Network(#[from] NetworkError),
+    #[error(transparent)]
+    NodeConfig(#[from] NodesConfigError),
+}
```

Comment:
As a convention, we _try_ and use the name `Error` only once per crate. This is often defined at the crate level and is meant to be a top-level public-facing error type. For other internal/intermediate errors (if we actually need them), we use a more descriptive name to avoid confusion at import time.